programs/rewards/Cargo.toml

```diff
@@ -22,3 +22,7 @@ crate-type = ["cdylib", "lib"]
 [[test]]
 name = "integration_tests"
 path = "tests/rewards/tests.rs"
+
+[features]
+no-entrypoint = []
+cpi = ["no-entrypoint"]
```

Comment:
Could you please elaborate on why we need an additional alias for the `no-entrypoint` feature?

Reply:
not really, just copy-pasted it here. We're not providing a cpi client/call-wrappers anyway. Will drop it